terminal/glint.go

```diff
@@ -11,6 +11,7 @@ import (
 	"strings"
 	"text/tabwriter"
 
+	"github.com/bgentry/speakeasy"
```

Comment:
Instead of using this library, can we give the term library a go instead (https://pkg.go.dev/golang.org/x/term#ReadPassword)?